File: mirai-core/src/main/java/net/mamoe/mirai/utils/LoggerTextFormat.java
package net.mamoe.mirai.utils;

/**
 * @author NaturalHG
 */
public enum LoggerTextFormat {
    RESET("\33[0m"),

    BLUE("\033[0;34m"),
    BLACK("\033[0;30m"),
    DARK_GREY("\033[1;30m"),
    LIGHT_BLUE("\033[1;34m"),
    GREEN("\033[0;32m"),
    LIGHT_GTEEN("\033[1;32m"),
    CYAN("\033[0;36m"),
    LIGHT_CYAN("\033[1;36m"),
    RED("\033[0;31m"),
    LIGHT_RED("\033[1;31m"),
    PURPLE("\033[0;35m"),
    LIGHT_PURPLE("\033[1;35m"),
    BROWN("\033[0;33m"),
    YELLOW("\033[1;33m"),
    LIGHT_GRAY("\033[0;37m"),
    WHITE("\033[1;37m");

    private String format;

    LoggerTextFormat(String format) {
        this.format = format;
    }

    @Override
    public String toString() {
        //if(MiraiServer.getInstance().isUnix()){
        return format;
        // }
        // return "";
    }
}

File: mirai-core/src/commonMain/kotlin/net.mamoe.mirai/utils/LoggerTextFormat.kt
package net.mamoe.mirai.utils

/**
 * @author NaturalHG
 */
enum class LoggerTextFormat(private val format: String) {
    RESET("\u001b[0m"),

    BLUE("\u001b[0;34m"),
    BLACK("\u001b[0;30m"),
    DARK_GREY("\u001b[1;30m"),
    LIGHT_BLUE("\u001b[1;34m"),
    GREEN("\u001b[0;32m"),
    LIGHT_GTEEN("\u001b[1;32m"),
    CYAN("\u001b[0;36m"),
    LIGHT_CYAN("\u001b[1;36m"),
    RED("\u001b[0;31m"),
    LIGHT_RED("\u001b[1;31m"),
    PURPLE("\u001b[0;35m"),
    LIGHT_PURPLE("\u001b[1;35m"),
    BROWN("\u001b[0;33m"),
    YELLOW("\u001b[1;33m"),
    LIGHT_GRAY("\u001b[0;37m"),
    WHITE("\u001b[1;37m");

    override fun toString(): String {
        //if(MiraiServer.getInstance().isUnix()){
        return format
        // }
        // return "";
    }
}
